tools/tune/eq/eq_fir_blob_pack.m

```diff
@@ -86,7 +91,7 @@
 nbytes_data = nb16 * 2;
 
 %% Get ABI information
-[abi_bytes, nbytes_abi] = eq_get_abi(nbytes_data);
```

Comment:
Is it necessary to have `bs `output at lines [39,47] on the console?
Otherwise, you may delete.

Reply:
Not really, I'll delete it.